tests/pkcli/github_test.py

```diff
@@ -19,6 +19,36 @@
 )
 
 
+def test_repo_no_wiki():
+    from pykern import pkconfig
+    from pykern.pkdebug import pkdlog, pkdp, pkdc, pkdexc
+
+    pkconfig.reset_state_for_testing(
+        {
+            "PYKERN_PKCLI_GITHUB_TEST_MODE": "1",
+            "PYKERN_PKCLI_GITHUB_API_PAUSE_SECONDS": "0",
+        }
+    )
+    import os
+    from pykern.pkcli import github
+    from pykern import pkunit
+    from pykern import pkio
+    import subprocess
+
+    with pkunit.save_chdir_work():
+        github.backup()
+
+    pkunit.pkeq(
```

Comment:
I think this can be simplified and made clearer what we are testing:
```python
    for n, c in PKDict(
        {"radiasoft-test-pykern-github": 1, "radiasoft-test-pykern-github-no-wiki": 0}
    ).items():
        pkunit.pkeq(c, len(pkio.sorted_glob(f"**/{n}.wiki.git/config")))
```

Reply:
After running once I realized it would be good to know which repo failed. So changing to something like
` pkunit.pkeq(c, len(pkio.sorted_glob(f"**/{n}.wiki.git/config")), "repo={}", n)`

Reply:
Also, this block should be indented under the with so only the work dir is searched.